pkg/alertmanager/operator.go

```diff
@@ -838,11 +840,6 @@ func (c *Operator) provisionAlertmanagerConfiguration(ctx context.Context, am *m
 		return nil
 	}
 
-	amConfigs, err := c.selectAlertmanagerConfigs(ctx, am, version, store)
```

Comment:
I moved this to later step so that `cfgBuilder.initializeFromAlertmanagerConfig()` will populate global config in config builder object